Makefile

```diff
@@ -9,6 +9,7 @@ K8S_DASHBOARD_PROXY_PORT := 30082
 MINIO_PROXY_PORT := 30084
 FLYTE_SANDBOX_NAME := flyte-sandbox
 KUBE_CONFIG := ~/kubeconfig
+FLYTE_CONFIG := ~/.flyte/config.yaml
```

Comment:
I feel like for purposes of flytesnacks, this shouldn't touch things outside of this directory.  What if the user had a file here that wasn't backed up or something?

Reply:
we need a file to point flytectl and I also understand your concern that why's I changed the name of config